src/components/userAuth/SignIn.js

```diff
@@ -166,3 +167,15 @@ export default function SignIn() {
     </Container>
   );
 }
+
+SignIn.propTypes = {
+  apiUrl: PropTypes.string,
+  errorHandler: PropTypes.func,
+  submit: PropTypes.func,
+  navigate: PropTypes.func,
+  signOut: PropTypes.func,
+  status: PropTypes.string,
+  handleChange: PropTypes.func,
+  handleLoginError: PropTypes.func,
+  handleSubmit: PropTypes.func,
+};
```

Comment:
Same with this component; we can remove all the changes in this file since it's not expecting any props.

Reply:
Thank you , commit a180c241a567fc4777440760cc7d1d8f38f1a5b3